frontend/src/features/navigation/data/steps.ts

```diff
@@ -20,9 +20,19 @@ const steps: Step[] = [
     path: "results",
   },
   {
-    name: "Performance",
+    name: "Initial Data",
```

Comment:
We should change it to Range & Endurance

Reply:
Applied.